src/gridstack.js

```diff
@@ -1272,8 +1272,8 @@
 
         el.addClass(this.opts.itemClass);
         var node = self.grid.addNode({
-            x: el.attr('data-gs-x'),
-            y: el.attr('data-gs-y'),
+            x: parseInt(el.attr('data-gs-x'), 10),
+            y: parseInt(el.attr('data-gs-y'), 10),
             width: el.attr('data-gs-width'),
```

Comment:
is there a reason you are not doing width/height/maxwidth/etc... as int as well ? el.attr() returning string should be checked throughout code IMO.